classes/abstracts/ActionScheduler.php

```diff
@@ -180,6 +180,24 @@ public static function init( $plugin_file ) {
 		add_action( 'action_scheduler/migration_complete', 'ActionScheduler_WPCommentCleaner::maybe_schedule_cleanup' );
 	}
 
+	/**
+	 * Issue deprecated warning if an Action Scheduler function is called in the shutdown hook.
+	 *
+	 * @param string $function_name The name of the function being called.
+	 */
+	public static function check_shutdown_hook( $function_name ) {
+		if ( 'shutdown' == current_filter() ) {
```

Comment:
Nitpick - linters in our other projects would complain about not using strict equals here.